web/tests/e2e/specs/participantSetup.feature

```diff
@@ -1,9 +1,10 @@
 Feature: Participant Setup
 
+# Note that researcher chooses their own participant id. Unique code is generated by the app 
 Scenario: Setup participant on server
     Given The researcher is logged in to the web interface
     When They select the "Add participant" option
-    Then They will be able to enter a participant id of their choosing and a unique code (generated by the app)
-    And The participant account will be created and stored in the database
+    Then They will be able to enter participant details
+    And The details shall include a participant id and a unique code
```

Comment:
These two lines described what's the "Add participant" form like.
I think another operation should be added "The research input the participant id and unique code, and click button "Confirm""

Reply:
Done, with slightly different wording